packages/Tokens/src/tokens.json

```diff
@@ -45,14 +45,14 @@
     "orangeLighten40": "#fae7e3",
     "orangeDisabled": "#e99c8e",
     "orangeHighlight": "#e1624b",
-    "purple": "#53529f",
-    "purpleDarken10": "#41417d",
-    "purpleDarken20": "#302f5c",
-    "purpleLighten10": "#7575af",
-    "purpleLighten20": "#9695c2",
-    "purpleLighten30": "#b6b5d5",
-    "purpleLighten40": "#d6d6e7",
-    "purpleLighten50": "#f6f6fa",
+    "galvanize": "#785cba",
```

Comment:
is there a reason why `galvanize` is the name for the variable?
@mikrotron @nahumzs

Reply:
I think we should have a quick discussion about this. Eventually we will be replace all purple color hex values, but keeping the same token names, hence mine and Mike P's decision to use `$color--purple-new` as the token name for the new Galvanize purple.

Reply:
I'm not entirely sure, to me, we shouldn't use the brand name for naming our tokens, they should be agnostic with their name? any specific to use this name @mikrotron ?

Reply:
ahh my mistake, i believe i was to leave the purple tokens as "purple" not "galvanize". Should i rename them as #color-purple-new or keep $color-purple?

Reply:
The approach in the [`acl-ui` PR](https://github.com/acl-services/acl-ui/pull/1182) is what we should go with here too.  Let's just update what `$purple` means.

Reply:
@nahumzs, @johnpeele are you both more comfortable with this new approach (same approach in `acl-ui` too - https://github.com/acl-services/acl-ui/pull/1182)?

Reply:
:/ not thrill about this approach but I guess $purple is ok

Reply:
I'm just not sure what is going on, what is the purpose of this ticket/PR?

Reply:
@johnpeele, this PR will update all of the `$purple` tokens to be the new Galvanize branded purple tones.  We won't need to change CSS throughout our apps*, just update `acl-ui` (well, _this_ PR is more to keep `paprika` in sync with that change).  

* This isn't 100% true.  Only about 86% true.

Reply:
@mikrotron First, love the percentage humor, well done.

Second, If we are only updating hex values and not token names then I am 110% on board with this. 👍🏼

Third, now Nahum's last comment in this thread is even more confusing to me. Why are you not thrilled about this approach @nahumzs ?